pkg/qemu/qemu.go

```diff
@@ -78,7 +78,7 @@ func EnsureDisk(cfg Config) error {
 	}
 	args := []string{"create", "-f", "qcow2"}
 	if !isBaseDiskISO {
-		args = append(args, "-b", baseDisk)
+		args = append(args, "-F", "qcow2", "-b", baseDisk)
```

Comment:
Thx

question: does -F qcow2 work when the base image is non-qcow2 (especially raw)?

Reply:
I don't know, but I suspect it won't. :(

Reply:
Can we inspect the format?

Reply:
opened https://github.com/lima-vm/lima/pull/176 for detecting the format